src/mainPublicShareSidebar.js

```diff
@@ -100,21 +80,27 @@ function addTalkSidebarTrigger() {
 addTalkSidebarTrigger()
 
 /**
- *
+ * Mount the Talk sidebar next to the main content.
  */
-function getShareToken() {
-	const shareTokenElement = document.getElementById('sharingToken')
-	return shareTokenElement.value
+function addTalkSidebar() {
+	// location.pathname is expected to be like /index.php/s/<shareToken>
+	const shareToken = window.location.pathname.split('/').pop()
```

Comment:
You can find it in initial state
![image](https://github.com/user-attachments/assets/55455594-6ebe-4805-8776-80a863dcf6dd)